add new UNC survey and labels

mattwigway opened the pull request:
This adds trip label configs for the `unc-ebike` OpenPATH deploy, as well as a custom onboarding survey.

We would ideally like to display the replaced mode choice only when the selected mode is e-bike, but I wasn't sure how to specify that in the config. Our onboarding survey is trivial because we have a much more extensive survey where participants can choose at the end whether to participate in OpenPATH data collection, so we'll be generating opcodes within that survey so we can link OpenPATH to external survey responses, to avoid having people re-enter the same information. My understanding is that it is not possible to remove the onboarding survey completely, so we added a trivial survey.

I haven't tested any of this, I wasn't sure how to test configs locally, but if that's possible I'd be happy to do local testing.

shankari commented:
@mattwigway 

> We would ideally like to display the replaced mode choice only when the selected mode is e-bike, but I wasn't sure how to specify that in the config. 

It is already specified here
https://github.com/e-mission/nrel-openpath-deploy-configs/pull/119/files#diff-60c020caa0849b84d7f3fa41a419accb60b2fac560e1d7af3582c230188fe0b8L20

> My understanding is that it is not possible to remove the onboarding survey completely, so we added a trivial survey.

Sure, makes sense

> I haven't tested any of this, I wasn't sure how to test configs locally, but if that's possible I'd be happy to do local testing.

Your environment is specific to your program, so feel free to edit and test as many iterations as needed.
Just make sure to test with an opcode from the "test" subgroup - e.g. generated using 
https://unc-ebike-openpath.stratus.nrel.gov/join/?sub_group=test

In the admin dashboard, you can get filter by subgroup. So you can filter out all the test data before your analysis.
So you can have multiple test phones, label trips as e-bike when they were not, change surveys and labels multiple times,...

mattwigway commented:
@shankari Thanks! I have updated `mode_studied` to match the way it is spelled in our labels file, and switched to pre-generated opcodes in the config per our email discussion. I think this is ready for merge.

shankari commented on label_options/unc-ebike-label-options.json:
```diff
@@ -0,0 +1,100 @@
+        "no_travel": "Would not have made trip"
+      },
+      "es": {
+        "car": "Car",
+        "walk": "Walk",
+        "bike": "Regular bike",
+        "ebike": "E-bike",
+        "bus": "Bus",
+        "taxi": "Taxi",
+        "ridehail": "Uber / Lyft",
+        "train": "Train / Amtrak",
+        "air": "Airplane",
+        "not_a_trip": "Not a trip",
+        "other": "Other",
+        "home": "Home",
+        "work": "Work",
```
Shouldn't this be in Spanish?

mattwigway replied:
Yes, we are in the process of hiring a translator so this is just a placeholder for now

shankari commented on survey_resources/unc-ebike/onboarding-v1.xml:
As you can see from the admin dashboard, we detect this automatically. I understand that you have a placeholder survey, but maybe you could have an interesting question here like "what do you expect to get out of this data collection" with potential answers like "understand my carbon footprint" or "contribute to better understanding of the program" or ....
